central/detection/lifecycle/manager_impl.go

```diff
@@ -201,7 +201,7 @@ func (m *managerImpl) flushIndicatorQueue() {
 	defer centralMetrics.SetFunctionSegmentDuration(time.Now(), "CheckAndUpdateBaseline")
 
 	// Group the processes into particular baseline segments
-	baselineMap := make(map[processBaselineKey][]*storage.ProcessIndicator)
+	baselineMap := make(map[processBaselineKey][]*storage.ProcessIndicator, len(indicatorSlice))
```

Comment:
I don't agree with this change. indicatorSlice can be much bigger than the resulting map so this could result in way over provisioning. I would like to see that map allocations are slow before allocating so much